<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="spaceLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Space</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="unitsLabel">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Units</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="modelLabel">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Model</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="parameter1Label">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Parameter 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="parameter2Label">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Parameter 2</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QComboBox" name="spaceComboBox">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <item>
              <property name="text">
               <string>Select Space...</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2D</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3D</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="unitsComboBox">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <item>
              <property name="text">
               <string>Select Units</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>m</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mm</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="modelComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>2</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <item>
              <property name="text">
               <string>Select Model</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>U</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>B</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>T</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="parameter1LineEdit">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="placeholderText">
              <string>Parameter 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="parameter2LineEdit">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="placeholderText">
              <string>Parameter 2</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="orientationPictureLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Select A Model to Display the Corresponding Image</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
